packages/metrics/tests/unit/Metrics.test.ts

```diff
@@ -104,15 +104,15 @@ describe('Class: Metrics', () => {
 
       expect(console.log).toBeCalledTimes(2);
       expect(loggedData[0][dimensionItem.name]).toEqual(dimensionItem.value);
-      expect(loggedData[0]._aws.CloudWatchMetrics[0].Dimensions[0].length).toEqual(1);
+      expect(loggedData[0]._aws.CloudWatchMetrics[0].Dimensions[0].length).toEqual(2);
       expect(loggedData[1][dimensionItem.name]).toBeUndefined();
-      expect(loggedData[1]._aws.CloudWatchMetrics[0].Dimensions[0].length).toEqual(0);
+      expect(loggedData[1]._aws.CloudWatchMetrics[0].Dimensions[0].length).toEqual(1);
     });
 
     test('Adding more than max dimensions should throw error', () => {
       expect.assertions(1);
       const metrics = new Metrics();
-      for (let x = 0; x < MAX_DIMENSION_COUNT; x++) {
+      for (let x = 1; x < MAX_DIMENSION_COUNT; x++) {
```

Comment:
The `service` dimension is always set now, so we need to start counting from 1 instead of 0.

Reply:
Let's add this as comment above this line please. It's not immediately clear unless you know how it works.

Reply:
Done